From d3bdd384a18edaf90eab0d387a08beb8e7044f62 Mon Sep 17 00:00:00 2001
From: Chun-wei Fan <fanchunwei@src.gnome.org>
Date: Fri, 11 Nov 2016 21:03:46 +0800
Subject: gdkscreen-win32.c: Also define _WIN32_WINNT

As in the last commit on gdkdisplay-win32.c, we need to define that to be
0x0600 (Vista) or later so that the items needed in the Windows headers be
activated.

See: https://bugzilla.gnome.org/show_bug.cgi?id=768081#c62
---
 gdk/win32/gdkscreen-win32.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/gdk/win32/gdkscreen-win32.c b/gdk/win32/gdkscreen-win32.c
index 3c14227..281b680 100644
--- a/gdk/win32/gdkscreen-win32.c
+++ b/gdk/win32/gdkscreen-win32.c
@@ -16,6 +16,9 @@
  */
 
 #include "config.h"
+
+#define _WIN32_WINNT 0x0600
+
 #include "gdk.h"
 #include "gdkprivate-win32.h"
 #include "gdkscreenprivate.h"
-- 
cgit v0.12

